docs/datasync/datasync-soft-delete.md

```diff
@@ -61,22 +74,17 @@ type CommentDeltaList {
   items: [CommentDelta]!
   lastSync: String
 }
-
-type Query {
-  ...
-  syncComments(lastSync: String!, filter: CommentFilter): CommentDeltaList!
-}
 ```
 
- The `@delta` annotation adds a `sync` query or a delta query as shown above. This allows you to get all the changed(updates and deletes) documents in a collection since the `lastSync` timestamp. Internally this uses the `updatedAt` database field to check if any documents in the database have been modified, by comparing client provided `lastSync` timestamp value. The documents to be returned in a `sync` query may also be filtered, so you can, for example, only `sync` comments on a specific post.
+The `Delta` type for a model consists of all of the model's transformed properties, as well as a `_deleted` field which is used internally to mark documents as deleted in the database. Thus `delete` mutations only mark documents with `_deleted: true` instead of actually removing them.
 
-The `@delta` annotation also adds a `_deleted` field to the `delta` type that tracks if a document has been deleted. Thus `delete` mutations only mark documents with `_deleted: true` instead of actually removing them.
+The `DeltaList` is a container for `Delta` type, which also returns a `lastSync` timestamp, that can be used in a subsequent `sync` query.
 
 > **NOTE**: `Soft Deletes` strategy can only get you the latest version of changed documents ignoring any in-between states that may have transpired between `lastSync` and now.
 
 ### Modify the template to support Data Synchronization
 
-In the [`src/index.ts`]() file of the template, use  `DataSyncPlugin` and compliant data sources in `buildGraphbackAPI`:
+In the [`src/index.ts`](https://github.com/aerogear/graphback/blob/templates-0.14.0/templates/ts-apollo-mongodb-backend/src/index.ts) file of the template, use  `DataSyncPlugin` and compliant data sources in `buildGraphbackAPI`:
```

Comment:
Is there some way to use a link of the latest templates releases and avoid the versioned link e.g `-0.14.0`? doing so we probably wouldn't have to care about revising the fine grained details in docs when doing the upgrade.

Reply:
I think we might need minimal datasync template anyway :)

Reply:
I have changed it to point to master for the meantime, I realise that this is not optimal, but I don't see any other workarounds here except actually pushing a datasync template and linking to it, unless I am missing something :D

We could always remove this link, it's not exactly being that much more useful as it is